submissions/models.py

```diff
@@ -124,6 +123,25 @@ class Submission(models.Model):
     # name so it continues to use `raw_answer`.
     answer = JSONField(blank=True, db_column="raw_answer")
 
+    # Has this submission been soft-deleted? This allows instructors to reset student
+    # state on an item, while preserving the previous value for potential analytics use.
+    DELETED = 'D'
+    ACTIVE = 'A'  # Note that, for migration purposes, a status value of None will be handled as 'Active'.
+    # If you are adding more statuses, be *sure* to write a data migration first to handle this!
+    STATUS_CHOICES = (
+        (DELETED, 'Deleted'),
+        (ACTIVE, 'Active'),
+    )
+    status = models.CharField(max_length=1, choices=STATUS_CHOICES, null=True, default=None)
```

Comment:
@robrap This is the decision Christina was deferring to you. Using a null default here is arguably not the "right thing" to do (versus setting Active as the default), but it does provide a speedup in the outage-inducing migration (in tests, 90 seconds vs 60, when run with no load, see previous comments on this PR. Devops estimated about a 2x increase when running on the loaded prod server)